common/test/acceptance/pages/studio/overview.py

```diff
@@ -126,29 +128,42 @@ def in_editable_form(self):
         """
         Return whether this outline item's display name is in its editable form.
         """
+        # pylint: disable=no-member
```

Comment:
Most of the disables added here are done on a per-line basis; why the few per-method disables here?